homeassistant/components/ituran/entity.py

```diff
@@ -0,0 +1,49 @@
+"""Base for all turan entities."""
+
+from __future__ import annotations
+
+from pyituran import Vehicle
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import IturanDataUpdateCoordinator
+
+
+class IturanBaseEntity(CoordinatorEntity[IturanDataUpdateCoordinator]):
+    """Common base for Ituran entities."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: IturanDataUpdateCoordinator,
+        license_plate: str,
+    ) -> None:
+        """Initialize entity."""
+        super().__init__(coordinator)
+
+        self._license_plate = license_plate
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self._vehicle.license_plate)},
+            manufacturer=self._vehicle.make,
+            model=self._vehicle.model,
+            name=self._vehicle.model,
+            serial_number=self._vehicle.license_plate,
+        )
+
+    @property
+    def available(self) -> bool:
+        """Return True if vehicle is still included in the account."""
+        return self._license_plate in self.coordinator.data
+
+    @property
+    def _vehicle(self) -> Vehicle:
```

Comment:
but I don't think you use at all

Reply:
If you only use it for derived classes, it can just be an instance attribute, but properties are intended to be public

Reply:
It's not exactly an attribute since it get's the latest information from the coordinator (this was a suggestion from the previous code review).
What would be preferred? Switching it to be a public property just for the sake of code simplicity (not adding `()` when using this) of leaving it as private and make it a proper function for the derived classes to use/call?

Reply:
ah right, didn't look at the code again yesterday. I would just make it public, doesn't really matter too much

Reply:
Done